lzo.changes

```diff
@@ -1,3 +1,6 @@
+* Thu Nov 18 2021 Pekka Vuorela <pekka.vuorela@jolla.com> - 2.10-1
```

Comment:
I don't remember if this should have that `-1` or if it should be skipped but maybe worth to check. (Or just try what CI likes and if it doesn't then commit a fix on top.)

Additionally, I got a warning about a bogus date in changelog and that might also affect CI:
```
warning: bogus date in %changelog: Fri Apr  7 2005 Michael Schwendt <mschwendt[AT]users.sf.net>
```
But I don't recall if CI rejects packages just because of that.

Reply:
> I don't remember if this should have that -1 or if it should be skipped but maybe worth to check. (Or just try what CI likes and if it doesn't then commit a fix on top.)

Having the impression that it should be good here, version is the more standard rpm chagnes files.

> Additionally, I got a warning about a bogus date in changelog and that might also affect CI

Nice catch. Better fix that.

Reply:
The entries at the bottom seemed quite broken all. Removed the oldest broken ones, the 2005 content should not be relevant for us at all.